dotCMS/src/main/java/com/dotcms/rest/api/v1/content/ContentResource.java

```diff
@@ -822,4 +824,62 @@ private List<ExistingLanguagesForContentletView> getExistingLanguagesForContent(
+     * query, and leaving it to dotCMS itself. This endpoint uses the {@link ContentSearchForm} to
+     * define the search parameters, and returns the same {@link SearchView} object that the
+     * {@link com.dotcms.rest.ContentResource#search(HttpServletRequest, HttpServletResponse,
+     * boolean, SearchForm)} endpoint returns.
+     *
+     * @param request           The current instance of the {@link HttpServletRequest}.
+     * @param response          The current instance of the {@link HttpServletResponse}.
+     * @param contentSearchForm The {@link ContentSearchForm} object containing the search
+     *                          parameters.
+     *
+     * @return The {@link SearchView} object containing the search results.
+     *
+     * @throws DotDataException     An error occurred when interacting with the database.
+     * @throws DotSecurityException The User accessing this endpoint doesn't have the required
+     *                              permissions.
+     */
+    @POST
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    @Path("/search")
+    @Operation(operationId = "search", summary = "Retrieves content from the dotCMS repository",
+            description = "Abstracts the generation of the required Lucene query to look for user searchable fields " +
+                    "in a Content Type, and returns the expected results.",
+            tags = {"Content"},
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "The query has been executed. It's possible that " +
+                            "no contents matched the search criteria.",
+                            content = @Content(mediaType = "application/json",
+                                    schema = @Schema(implementation = SearchView.class)
+                            )
+                    ),
+                    @ApiResponse(responseCode = "400", description = "Bad request. Malformed JSON body"),
+                    @ApiResponse(responseCode = "401", description = "Invalid User, or not logged in"),
+                    @ApiResponse(responseCode = "403", description = "Forbidden"),
+                    @ApiResponse(responseCode = "500", description = "Internal Server Error")
+            }
+    )
+    public ResponseEntityView<SearchView> search(@Context final HttpServletRequest request,
+                             @Context final HttpServletResponse response,
+                             final ContentSearchForm contentSearchForm) throws DotDataException, DotSecurityException {
+        Logger.debug(this, () -> "Searching for contentlets with the following parameters: " + contentSearchForm);
+        final User user = new WebResource.InitBuilder(webResource)
+                .requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .requiredBackendUser(true)
```

Comment:
only BE wondering?

Reply:
Add some log

Reply:
It already has logging, line 870

Reply:
The reason why I set it to BE only is because that's how the legacy DWR code is working now. I can certainly change it to any user, but I'd be a little concerned about potential DOS attacks and similar.